README.md

```diff
@@ -1,4 +1,151 @@
-# Awesome NEAR [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+# Awesome NEAR
 
 A curated list of awesome NEAR frameworks, libraries, software and resources.
 
+## Contacts
+
+* [Website](https://near.org)
+* [Discord](https://near.chat)
+
+## Documentation
+
+* [NEAR Docs](https://docs.near.org/)
+* [NEAR Blog](https://near.org/blog/)
+* [NEAR Wiki](https://wiki.near.org/)
+* [NEAR Specs](https://nomicon.io/)
+
+## Libraries / Frameworks
+
+Smart Contract SDK
+
+* [near-sdk-rs](https://github.com/near/near-sdk-rs)
+* [near-sdk-js](https://github.com/near/near-sdk-js)
+* [near-sdk-as](https://github.com/near/near-sdk-as)
+
+Frontend Libraries
+
+* [near-api-js](https://github.com/near/near-api-js)
+* [near-api-go](https://github.com/textileio/near-api-go)
+* [near-api-rs](https://github.com/bicarus-labs/near-api-rs)
+* [near-api-py](https://github.com/near/near-api-py)
+* [near-api-swift](https://github.com/near/near-api-swift)
+
+Testing Tools
+
+* [near-workspaces-rs](https://github.com/near/workspaces-rs)
+* [near-workspaces-js](https://github.com/near/workspaces-js)
+
+Game Frameworks
+
+* [unity](https://github.com/near/near-api-unity)
```

Comment:
We also have another newer Unity SDK. will look for it later.

Reply:
I found [this repository](https://github.com/good1101/NearClientApi), is it this one?